preserve_model_gui.py

```
@@ -34,18 +35,18 @@
 
 @dataclass
 class AppConfig:
-    use_deployed: bool
+    """デプロイ済み関数を引くときの参照先。
+
+    Modal のコンテナ内で動く場合（`preserve_model.py::web`）は一時 App を
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)